src/shared/views/RecommendedCards.js

```diff
@@ -149,7 +151,19 @@ class _RecommendedCards extends React.Component<Props> {
 
     subscribeToProjects = () => {
         const { type, path, storeAs, ...options } = projectsQuery;
-        const { firebase, navigation } = this.props;
+        const { firebase, navigation, teamId } = this.props;
```

Comment:
If I get this right this overwrite the existing query to get the projects from firebase, but I have problems to find the file where we would query the projects normally. Can you point me to it?

I think it's fine, but just want to understand the initial query as well.

Reply:
Yes, this is the part that was modified to allow for "live" update of projects, so it's done in a slightly different way. On line 156 below, we call `firebase.watchEvent` which sets up a listener for changes from firebase. The details of the query are on line 91 above, in the same file. The `willFocus` and `willBlur` events are fired by the navigation when the user enters/leaves the screen with the list of projects.